idxmgmt/manager.go

```diff
@@ -201,11 +213,15 @@ func (m *manager) loadPolicy(ilmFeature feature, ilmSupporter libilm.Supporter)
 		return false, nil
 	}
 	policy := ilmSupporter.Policy().Name
+	if ilmSupporter.Policy().Body == nil {
```

Comment:
I don't think this is worth bloating the `ilm.Supporter` interface in `libbeat`.  

The name `IsPolicyLoaded` would also not be quite appropriate here, as this check decides wheter or not the `manager` responsible for loading policies should be called or not. We do use the standard ILM supporter, as all the loading logic is identic, and I don't want to diverge from that, just because of this check.

Reply:
if i would have to answer the question "how to know if an ilm policy is loaded" i don't think that "check that `ilmSupporter.Policy().Body` is `nil`" would cross my mind, but i'm fine with that. 
hopefully we don't have to do this check in many places